qemu/tests/openflow_acl_test.py

```diff
@@ -174,9 +174,8 @@ def access_service(access_sys, access_targets, disabled, host_ip,
                     test.fail(err_msg)
                 session.close()
 
-    def get_acl_cmd(protocol, in_port, action, extra_options):
+    def get_acl_cmd(protocol, action, extra_options):
```

Comment:
in_port is valid for drop actions.
but the value of in_port should be modified.
try this cmd in host: ovs-ofctl add-flow ovs0 tcp,in_port=$guest1_tap(eg: tap0),nw_dst=$guest2_ip,tp_src=23,actions=drop